galasa-ui/src/components/test-runs/TestRunsTable.tsx

```diff
@@ -183,6 +183,10 @@ export default function TestRunsTable({runsList,limitExceeded, visibleColumns, o
     router.push(`/test-runs/${runId}`);
   };
 
+  if (visibleColumns.length === 0) {
+    return <p>{translations('noColumnsSelected')}</p>;
+  }
```

Comment:
I think it would be worth adding a unit test to check that hiding all columns will display this message